examples/time_series/goes_xrs_nrt_data.py

```diff
@@ -64,9 +69,31 @@
 goes_ts = ts.TimeSeries(goes_data, meta, units, source="xrs")
 
 ###############################################################################
-# Finally, we can plot the timeseries.
+# NOAA also provides the past 7 days of flare event identifications which we 
+# can also parse.
+# Note that it is possible that no flares occurred in the last 7 days.
+
+flare_events = pd.read_json(
+    "https://services.swpc.noaa.gov/json/goes/primary/xray-flares-7-day.json"
+)
+
+###############################################################################
+# Next we sort the list and retain the five largest flares. 
+
+if len(flare_events) > 0:
+    largest_flares = flare_events.sort_values('max_xrlong', ascending=False)[:5]
+else:
+    largest_flares = []
+
+###############################################################################
+# Finally, we plot the timeseries and overlay the largest flares that occurred.
 
 fig, ax = plt.subplots()
 goes_ts.plot(axes=ax)
-
+if len(largest_flares) > 0:
```

Comment:
```suggestion
if largest_flares:
```
would this work?

Reply:
It would work, but I prefer the explicit call in a gallery example because I think falsy values are confusing to the average user